core/import-suunto.c

```diff
@@ -432,6 +432,21 @@ static int dm5_dive(void *param, int columns, char **data, char **column)
 	if (data[5])
 		utf8_string(data[5], &state->cur_dive->dc.model);
 
+	if (data[25]) {
+		//enum divemode_t {OC, CCR, PSCR, FREEDIVE, NUM_DIVEMODE, UNDEF_COMP_TYPE};	// Flags (Open-circuit and Closed-circuit-rebreather) for setting dive computer type
+		switch(atoi(data[25])) {
+			case 1:
+				state->cur_dive->dc.divemode = 0;
+				break;
+			case 5:
+				state->cur_dive->dc.divemode = 1;
+				break;
+			default:
+				state->cur_dive->dc.divemode = 0;
+				break;
```

Comment:
one could argue that '1' and 'default' are kinda the same...

Reply:
I like to be explicit on the information I have been able to dig out from the DM5 (or other) format. Now it is documented that 1 is OC, unlike if I was using the default for it.